src/main/java/de/tum/in/www1/artemis/domain/modeling/ModelElement.java

```diff
@@ -0,0 +1,74 @@
+package de.tum.in.www1.artemis.domain.modeling;
+
+import java.io.Serial;
+import java.io.Serializable;
+
+import javax.persistence.*;
+import javax.validation.constraints.Size;
+
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@Entity
+@Table(name = "model_element")
+@org.hibernate.annotations.Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class ModelElement implements Serializable {
+
+    @Serial
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @Size(min = 36, max = 36)
+    @Column(name = "modelElementId", unique = true, columnDefinition = "CHAR(36)", length = 36)
+    // Apollon Ids are exactly 36 characters long, Feedback.reference used this modelElementId to create a connection
+    private String modelElementId;
+
+    // TODO: we could also store the feedbackId here to simplify the check whether a model element has been corrected or not.
+
+    @Column(name = "modelElementType", nullable = false)
```

Comment:
I can see a different column name in the [...changelog.xml](https://github.com/ls1intum/Artemis/pull/3515/files#diff-f537f9510718b558a3b05dabda98d558d7c72aee44ff94a976b6fcc81fc75c5eR20) . Should they be the same?

Reply:
These changes were made by Stephan so I don't know :smile: